trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/loss-given-default.njk

```diff
@@ -5,6 +5,10 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle -%}
+  What’s the loss given default (LGD) for {{ deal.submissionDetails.supplierName}}?
```

Comment:
GEF deal?

Reply:
Yes, it is copied from H1, if it works bellow it works in pageTitle